src/actions/auth.js

```diff
@@ -0,0 +1,248 @@
+const StrangeAuth = require('strange-auth');
+const WebClient = require('../utils/web-client');
+const History = require('../wiring/history');
+const AuthAct = require('../action-types/auth');
```

Comment:
For utils, wiring, and action-types you shouldn't need to do relative paths here, I think you should be able to just do

const WebClient = require('utils/web-client');
const History = require('wiring/history');
const AuthAct = require('action-types/auth');

Reply:
I personally like seeing the relative paths (esp when we're only one level deep, and not way down in `routes/`), but that jus me.

Reply:
I agree with Devin 😛but can do whatever the group wants!

Reply:
Don't worry about making a change for that =)